src/utils/testingRender.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { render as rtlRender } from '@testing-library/react';
+import { configureStore } from '@reduxjs/toolkit';
+import { Provider } from 'react-redux';
+
+import PropTypes from 'prop-types';
+
+import reducer from 'state/reducers';
+
+function render(
```

Comment:
WDYT about moving this into a specific `/tests` folder since this is for testing purposes